config/locales/en.yml

```diff
@@ -2432,6 +2432,9 @@ en:
     Met: Met
     Unmet: Unmet
     NA: N/A
+  error_messages:
+    need_home_page_or_url: Need at least a home page or repository URL
+    the_form_contains: The form contains
```

Comment:
```yaml
  form_contains:
    one: The form contains 1 error.
    other: The form contains %{count} errors.
```
Here is an alternative to support more complicated pluralization rules, which I don't think we need to bother with: https://stackoverflow.com/questions/6166064/i18n-pluralization